server.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"app/middleware/auth"
```

Comment:
I see no easy way of doing this than using the fully qualified path (`github.com/impronunciable/hachdash-api/middleware/auth`)